container-host-files/etc/scf/config/scripts/patches/fix_gorouter_bpm_dns.sh

```diff
@@ -0,0 +1,34 @@
+#! /usr/bin/env bash
+
+# This is a temporary patch needed to make kube-dns work for BPM-managed jobs.
+# This will go away with the transition to the cf-operator, which will not require BPM anymore.
+
+set -e
+
+PATCH_DIR=/var/vcap/jobs-src/gorouter/templates
+SENTINEL="${PATCH_DIR}/${0##*/}.sentinel"
+
+if [ -f "${SENTINEL}" ]; then
+  exit 0
+fi
+
+patch -d "$PATCH_DIR" --force -p0 <<'PATCH'
+--- bpm.yml.erb
++++ bpm.yml.erb
+@@ -13,3 +13,11 @@ processes:
+     pre_start: /var/vcap/jobs/gorouter/bin/bpm-pre-start
+   capabilities:
+   - NET_BIND_SERVICE
++  unsafe:
++    unrestricted_volumes:
++    - path: /etc/hostname
++    - path: /etc/hosts
++    - path: /etc/hosts
++    - path: /etc/resolv.conf
++    - path: /etc/ssl
++    - path: /var/lib
```

Comment:
All of `/var/lib` seems a bit much; just `/var/lib/ca-certificates` should be enough.

Also, needs comments to say what these mounts are for.

(These comments apply to the other patches as well)

Reply:
+1 on comments. Restricting the mount will require another round of testing

Reply:
Mount change fixed by Jan with commit 5e020dbe79029da6d3c00d7bd02f23f6c23459e7.
Comments to come.

Reply:
See https://github.com/SUSE/scf/pull/1956